tracing-core/src/dispatcher.rs

```diff
@@ -675,6 +677,7 @@ impl State {
 // ===== impl DefaultGuard =====
 
 #[cfg(feature = "std")]
+#[cfg_attr(docsrs, doc(cfg(feature = "std")))]
```

Comment:
i don't _think_ the attribute is necessary on trait impls for types that only exist when a flag is set, only for the _definitions_ of those types.